content/docs/guides/continuous-delivery/octopus-deploy.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Octopus Deploy
+
+menu:
+    userguides:
+        parent: cont_delivery
+        weight: 1
+---
+
+[Octopus Deploy](https://octopus.com) is a deployment automation server, designed to make it easy to orchestrate releases and deploy applications, whether on-premises or in the cloud. Octopus is unlike a traditional CI system. In fact, Octopus is not a CI system. However, it can integrate with your existing build pipeline and integrate with several of your existing CI systems, like Jenkins, TeamCity, Azure DevOps etc.
```

Comment:
This sounds odd:
"Octopus is unlike a traditional CI system. In fact, Octopus is not a CI system. However, it can integrate..."

 How about,

"Unlike a traditional CI system, Octopus can integrate with your existing pipeline and integrate with your existing CI systems. Such as Jenkins, TeamCity, Azure DevOps, etc. Octopus does <specific thing separate from a prototypical CI/CD system>."

Reply:
Reading the intro paragraph now made me realize it can be a lot simpler than that. I'll update it to something much more simple.